test/blackbox-tests/test-cases/name-field-validation/run.t

```diff
@@ -2,7 +2,7 @@
   File "dune", line 3, characters 7-14:
   3 |  (name foo.bar)
              ^^^^^^^
-  Error: Invalid library name.
-  Hint: library names must be non-empty and composed only of the following
+  Error: "foo.bar" is an invalid library name
+  library names must be non-empty and composed only of the following
```

Comment:
Hmm, this should start with a capital letter and the previous line should end with a dot.
Not sure what's the right way to do this without also making some lines say:

> Hint: Library names must

Reply:
I'll try to remove the `Hint` bit. It's not really a hint if it's a hard requirement.

Reply:
Okay, I've gotten rid of hint fixed casing and periods.

By the way, I just noticed that the formatting of some of those messages looks a bit goofy. There's a line break that seems unnecessary. Oh well 😂

Reply:
Looks good.